onnx/defs/experiments/defs.cc

```diff
@@ -7,6 +7,91 @@ using namespace onnx;
 using AttrType = onnx::OpSchema::AttrType;
 using SupportType = onnx::OpSchema::SupportType;
 
+OPERATOR_SCHEMA(Identity)
```

Comment:
still, why on earth do you need identity? :) isn't it just an edge in the graph?